_issues/rapist.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Didn't she get a rapist acquitted, and then laugh about the girl who was raped?"
+categories: controversy
+---
+
+**Q:** Didn't Hillary get a rapist acquitted, and then laugh about the girl who was raped?  
+
+**A**: In the second Presidential debate, Donald Trump claimed "Her client she represented
+got him off and she is seen laughing on two occasions laughing at the girl who was raped." This is [completely false](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt). Here are the facts:
```

Comment:
Agree. It's a pretty persistent claim that comes up every so often. I think "repeatedly disproven" sounds more effective.